File: CH1-input-process-output/EX2-sales-prediction/salesPrediction.java
import java.util.Scanner;

public class salesPrediction {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int profit;
        float totalSales;
        
        // Input
        System.out.print("Enter a total sales : $");
        totalSales = input.nextInt();

        // Process
        double temp  = totalSales * 0.23;
        profit = (int) temp;

        // Output
        System.out.println("Profit : $" + profit);
    }
}

File: CH1-input-process-output/EX2-sales-prediction/source-code/salesPrediction.cpp
#include <iostream>
using namespace std;

int main()
{
    float profit;
    float totalSales;

    // Input
    cout << "Enter a total sales : $";
    cin >> totalSales;

    // Process
    profit = totalSales * 0.23;

    // Output
    cout << "Profit : $" << profit << endl;

    return 0;
}